isisd/isis_csm.c

```diff
@@ -74,6 +75,14 @@ isis_csm_state_change(int event, struct isis_circuit *circuit, void *arg)
 	case C_STATE_NA:
 		if (circuit)
 			zlog_warn("Non-null circuit while state C_STATE_NA");
+		ifp = (struct interface *)arg;
+		isis = isis_lookup_by_vrfid(ifp->vrf_id);
+		if (isis == NULL) {
+			zlog_warn(" %s : ISIS routing instance not found",
```

Comment:
nit: `zlog_warn(%s: ISIS ...` (please remove the extra white spaces)

Reply:
This is resolved.